frontend/src/components/pages/CampCreation/ScheduleSessions/SessionSidePanel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Box } from "@chakra-ui/react";
+import { CreateCampSession } from "../../../../types/CampsTypes";
+import AddSessionsForm from "./AddSessionsForm";
+import CurrentSessionsView from "./CurrentSessionsView";
+
+type SessionSidePanelProps = {
+  scheduledSessions: CreateCampSession[];
+  setScheduledSessions: (sessions: Array<CreateCampSession>) => void;
+};
+
+const SessionSidePanel: React.FC<SessionSidePanelProps> = ({
```

Comment:
nit: I know we're inconsistent with this in the repo, but why this vs. `React.ReactElement`?

Reply:
o mb i just got used to doing this at work - will fix